commit

```diff
@@ -2,15 +2,19 @@
 
 echo 'Running commit script'
 
-IGNORED_BRANCHES=("dev" "master" "qa" "uat" "staging")	
-IGNORED_PATH="./.ignore"
+IGNORED_BRANCHES=("dev" "master" "qa" "uat" "staging")
+
+ROOT_DIRECTORY_CMD="git rev-parse --show-toplevel"
+GIT_ROOT_DIRECTORY=$(eval $ROOT_DIRECTORY_CMD)
+
+CUSTOM_IGNORED_PATH="$GIT_ROOT_DIRECTORY/.smart-commit-ignore"
```

Comment:
why is CUSTOM_IGNORED_PATH only declared and not used?

Reply:
Lol, is it? I think I tested it myself. This might be a blunder. 😞

Reply:
I think your test passed because we have fallback array of branches 😂